### PR TITLE
CONTRIBUTING.md: fix typo & add clonning section

### PR DESCRIPTION
**Pull request by ekalinin**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -11,11 +11,22 @@ $ go version
...
 ```
 **Note:** If using gotip and `go version` does not return version 1.18, you may need to symbolic link `gotip` to `go`.
-```
-ls -s path_to_gotip /user/local/bin/go
+```sh
+$ ln -s `which gotip` /user/local/bin/go
 ```
 Use `which gotip` to get the path to gotip.
 
+## Clonning
```
Comment (OpenSauce) [inline CONTRIBUTING.md:19]:
```suggestion
## Cloning the Repository
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -11,11 +11,22 @@ $ go version
...
+
+After forking FerretDB you can clone for best practices use the following instruction:
```
Comment (OpenSauce) [inline CONTRIBUTING.md:21]:
```suggestion
After forking FerretDB you can clone the repository - for rgw best practices use the following instructions:
```

Just slight grammar fix, hope you don't mind :)

Comment (ekalinin) [reply CONTRIBUTING.md:21]: Sure, thanks! :)

**Review by OpenSauce — APPROVED** [review]

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -11,11 +11,22 @@ $ go version
...
-```
-ls -s path_to_gotip /user/local/bin/go
+```sh
+$ ln -s `which gotip` /user/local/bin/go
 ```
 Use `which gotip` to get the path to gotip.
 
+## Cloning the Repository
+
+After forking FerretDB you can clone the repository - for rgw best practices use the following instructions:
```
Comment (OpenSauce) [inline CONTRIBUTING.md:21]:
```suggestion
After forking FerretDB you can clone the repository - for the best practices use the following instructions:
```

@ekalinin I typed in “the" wrong 😄

**Review by AlekSi — APPROVED** [review]

**Pull request merged**